src/sidepanel/style.css

```
@@ -210,6 +210,14 @@ body {
   cursor: not-allowed;
 }
 
+#send-btn.cancel {
+  background: #cc3333;
```

Stylelint (17.2.0)

[error] Expected "#cc3333" to be "#c33" (color-hex-length)

(color-hex-length)